packages/jest-runtime/src/index.ts

```diff
@@ -950,16 +957,17 @@ class Runtime {
 
   private _execModule(
     localModule: InitialModule,
+    moduleSource: string | undefined,
```

Comment:
Name should probably indicate that this is only for modules explicitly compiled, not on the regular code path where the module is actually loaded from disk

Reply:
How about `runtimeModuleSource`?